### PR TITLE
Updated to include Account and team setup events, system console events

### PR DESCRIPTION
**Pull request by adamjclarkson**
Covering changes related to https://github.com/mattermost/mattermost-webapp/pull/5049 and https://github.com/mattermost/mattermost-webapp/pull/5047

### CODE REVIEW

**Comment by amyblais** [comment]
@adamjclarkson For future PRs for v5.22, please help submit a new PR in the ``5.22-documentation`` branch.

**Comment by adamjclarkson** [comment]
whoops ok sorry Amy

File: source/administration/telemetry.rst [hunk 1]
```diff
@@ -77,7 +77,7 @@ Non-personally Identifiable Error Information, distinguished by end users and Sy
 Non-personally Identifiable Diagnostic Information, distinguished by end users and System Admins
   Boolean when the following events occur:
 
-  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion
+  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion. Click events on signup_email, select_team, display_name and team_url pages.
```
Comment (jasonblais) [inline source/administration/telemetry.rst:80]: The events are actually already included
 - Account creation via email
 - team creation page view, team name and URL entry, team creation completion

We can add `select_team` event though. So perhaps:

Comment (jasonblais) [reply source/administration/telemetry.rst:80]:
```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, team selection page.
```

File: source/administration/telemetry.rst [hunk 2]
```diff
@@ -89,6 +89,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams 
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system.  Scheme ID; Team Admin Permissions; Team User Permissions; Channel Admin Permissions; Channel User Permissions; Number of teams the scheme is associated with
   - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu itesm in the left navigation.
```
Comment (jasonblais) [inline source/administration/telemetry.rst:92]: Is this for discovering the System Console? Or is this just for the menu items?

File: source/administration/telemetry.rst (hunk 1 above)
Comment (adamjclarkson) [reply source/administration/telemetry.rst:80]: Well the change is to track clicks on all form elements, buttons, textboxes and links on these pages, not just the page views.

File: source/administration/telemetry.rst (hunk 2 above)
Comment (adamjclarkson) [reply source/administration/telemetry.rst:92]:
```suggestion
  - *System Console Discovery Diagnostics:* click events on the hamburger menu and menu items: Administrator's Guide, Troubleshooting Forum, Commerical Support, About Mattermost; click events on the menu items in the left navigation.
```

Comment (adamjclarkson) [reply source/administration/telemetry.rst:92]: Just for the menu items

File: source/administration/telemetry.rst (hunk 1 above)
Comment (jasonblais) [reply source/administration/telemetry.rst:80]:
```suggestion
  - *Team and Account Setup Diagnostics:* Account creation via email, invite or UI, account creation page view, account creation completion; tutorial step and tip completion or opt out, team creation page view, team name and URL entry, team creation completion, clicks on all form elements, buttons, textboxes and links on sign up page, team selection page and team creation pages
```

Comment (jasonblais) [reply source/administration/telemetry.rst:80]: Got it, how about this?

File: source/administration/telemetry.rst (hunk 2 above)
Comment (jasonblais) [reply source/administration/telemetry.rst:92]: Just minor tweak:

Comment (jasonblais) [reply source/administration/telemetry.rst:92]:
```suggestion
  - *System Console Menu Discovery Diagnostics:* Clicks on the hamburger menu items of the System Console, including Administrator's Guide, Troubleshooting Forum, Commercial Support, About Mattermost, and clicks on the left-hand side navigation menu items
```

File: source/administration/telemetry.rst (hunk 1 above)
Comment (adamjclarkson) [reply source/administration/telemetry.rst:80]: yep wfm

**Pull request merged**